### PR TITLE
invalid URL giving console error fix.

### LINKED ISSUE #653: Invalid URL changes should redirect to home component
**Issue/Feature Description:**
Invalid URL changes should redirect to the home component.
**Why this issue to fixed / feature is needed(give scenarios or use cases):**
suppose that you directly changed the URL in the search box. In case it is a wrong URL we get a console error. 
**How to reproduce, in case of a bug:**
search any URL in the search box.
**Other Notes /  Environment Information: (Please give the env information, log link or any useful information for this issue)**
Screenshot (13)

SODACODE2022

### PR DESCRIPTION
**Pull request by ravisaxena23**
**What type of PR is this?**
 /kind bug fix

**What this PR does / why we need it**:
suppose that you directly changed the URL in the search box. In case it is a wrong URL we get a console error. so it needs to be fixed. added another routing path that redirects to the home component if invalid is given.
**Which issue(s) this PR fixes**:
Fixes #653 

**Test Report Added?**:
 /kind TESTED

**Test Report**:
Screenshot (16)
Screenshot (17)

**Special notes for your reviewer**:

### CODE REVIEW

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -50,6 +50,7 @@ const routes: Routes = [
     {path: 'storageDetails/:storageId', loadChildren: './business/delfin/storage-details/storage-details.module#StorageDetailsModule'},
     {path: 'modifyStorage/:storageId', loadChildren: './business/delfin/modify-storage/modify-storage.module#ModifyStorageModule'},
     {path: 'servicePlanManagement', loadChildren: './business/service-plan/service-plan.module#ServicePlanModule'},
+    {path: '**', redirectTo: '/home'}
```
Comment (anvithks) [inline src/app/app-routing.module.ts:53]: Redirection works but the css in the page is not changed.
When we are in any other page than the home page we have  white page background. 
On the home page we have a blue image background. 
When we redirect to home page in the code using the routerLink to navigate method we explicitly change this value and change the background. 
This check will need to be made in this case as well.

**Review by anvithks — CHANGES_REQUESTED** [review]
Please check the comment. We need to ensure non home page to home page transitions maintains the styling.

File: src/app/app-routing.module.ts (hunk 1 above)
Comment (ravisaxena23) [reply src/app/app-routing.module.ts:53]: this is a known issue on the soda dashboard. I checked without this branch this problem is still arising. However, this PR does not fix that this PR only fixes #653. for fixing it we can open a new PR.

**Review by anvithks — APPROVED** [review]
LGTM

**Comment by anvithks** [comment]
#SODA-PR-ACCEPTED

**Review by joseph-v — APPROVED** [review]
LGTM

**Pull request merged**